command/agent/http.go

```diff
@@ -783,10 +783,17 @@ func parseWait(resp http.ResponseWriter, req *http.Request, b *structs.QueryOpti
 }
 
 // parseConsistency is used to parse the ?stale query params.
-func parseConsistency(req *http.Request, b *structs.QueryOptions) {
+func parseConsistency(resp http.ResponseWriter, req *http.Request, b *structs.QueryOptions) {
 	query := req.URL.Query()
-	if _, ok := query["stale"]; ok {
-		b.AllowStale = true
+	if staleVal, ok := query["stale"]; ok {
+		if staleVal[0] == "true" || staleVal[0] == "" {
```

Comment:
Instead of requiring `"true"` or empty string, what if we use `strconv.ParseBool` here; that will help with compatibility with callers using other languages, etc.

Reply:
Thanks you for the suggestion and the super quick review. I have a question though. If it is empty string `strconv.ParseBool` will return `false` as well right :thinking: Just want to confirm on my understanding

Reply:
It will return false yeah, so we'll need to first explicitly check for empty string and interpret that as true